pycytominer/cyto_utils/cells.py

```diff
@@ -96,6 +108,9 @@ def __init__(
         self.is_subset_computed = False
         self.compartments = compartments
         self.compartment_linking_cols = compartment_linking_cols
+        self.fields_of_view = fields_of_view
+        self.fields_of_view_feature = "Metadata_Site"
```

Comment:
is this always going to be "Metadata_Site" - should we move to default __init__?

Reply:
Done - https://github.com/cytomining/pycytominer/pull/129/commits/2d27b5c0e91eb08215ef3bcea05c46cc51ad7e33